### PR TITLE
deprecate IP::World

### PR DESCRIPTION
**Pull request by DrHyde**
I'm the maintainer of [IP::World](https://metacpan.org/dist/IP-World), but it is now deprecated. This patch adds it to DiscouragedModules.

### CODE REVIEW

**Comment by Grinnz** [comment]
Thanks. The most useful alternative to present is probably [GeoIP2](https://metacpan.org/pod/GeoIP2) which is deprecated on CPAN but the free [GeoLite2 database](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data) is still updated so it works fine.

**Comment by DrHyde** [comment]
I would have recommended that if not for the deprecation notice! But if you know that it still works just fine (I've had no need to actually _do_ any geo-location in a while) then I can update the PR if you like.

**Comment by Grinnz** [comment]
It's a little roundabout to get the free database now: you have to create an account, and set up to auto-download using your license key and extract the file from a tarball, but I can confirm it still works.

File: lib/Perl/Critic/Policy/Community/DiscouragedModules.pm [hunk 1]
```diff
@@ -32,6 +32,7 @@ my %modules = (
 	'FindBin' => 'FindBin depends on the sometimes vague definition of "initial script" and can\'t be updated to fix bugs in old Perls. Use Path::This or lib::relative to work with the absolute path of the current source file instead.',
 	'HTML::Template' => 'HTML::Template is an old and buggy module, try Template Toolkit, Mojo::Template, or Text::Xslate instead, or HTML::Template::Pro if you must use the same syntax.',
 	'IO::Socket::INET6' => 'IO::Socket::INET6 is an old attempt at an IPv6 compatible version of IO::Socket::INET, but has numerous issues and is discouraged by the maintainer in favor of IO::Socket::IP, which transparently creates IPv4 and IPv6 sockets.',
+    'IP::World' => 'IP::World is deprecated as its are in one case discontinued, in the other no longer updated. Therefore its accuracy is ever-decrasing.',
```
Comment (Grinnz) [inline lib/Perl/Critic/Policy/Community/DiscouragedModules.pm:35]: I think there's a word missing here: "as its ??? are"

Also please mention the GeoIP2 alternative in this message as well (it's what will show up in the perlcritic output). I matched it to the documentation for most modules to keep it simple.

**Pull request merged**